vendor/github.com/prometheus/prometheus/tsdb/head.go

```diff
@@ -2076,106 +2076,90 @@ func (h *Head) updateWALReplayStatusRead(current int) {
 
 func GenerateTestHistograms(n int) (r []*histogram.Histogram) {
 	for i := 0; i < n; i++ {
-		h := histogram.Histogram{
-			Count:         10 + uint64(i*8),
-			ZeroCount:     2 + uint64(i),
-			ZeroThreshold: 0.001,
-			Sum:           18.4 * float64(i+1),
-			Schema:        1,
-			PositiveSpans: []histogram.Span{
-				{Offset: 0, Length: 2},
-				{Offset: 1, Length: 2},
-			},
-			PositiveBuckets: []int64{int64(i + 1), 1, -1, 0},
-			NegativeSpans: []histogram.Span{
-				{Offset: 0, Length: 2},
-				{Offset: 1, Length: 2},
-			},
-			NegativeBuckets: []int64{int64(i + 1), 1, -1, 0},
-		}
+		h := GenerateTestHistogram(i)
```

Comment:
Weird to see function `GenerateTestHistogram` in head.go, since it looks like code used for tests.